src/config/App.vue

```diff
@@ -218,9 +218,9 @@ ion-app {
 .menu > * {
   margin-right: 0.3em;
 }
-.wrap {
+/*.wrap {
```

Comment:
this had the purpose of centering the content on ultra-wide screens - is this still the case if you disable it? Could you provide screenshots of width > 1280++ ? I think you might have to change the number instead of just disabling it entirely...

Reply:
Okay, will do this

Reply:
![1990-width-center](https://user-images.githubusercontent.com/78471590/111656240-57eeaf00-8830-11eb-9b5e-c004a9803e9a.PNG)
![1280-width](https://user-images.githubusercontent.com/78471590/111656243-59b87280-8830-11eb-9e87-23cb6f6aedd7.PNG)

These two screens just for reference. Things working fine without entering that element.

Reply:
@ramandeepR try the big screen and then scroll to the image ... it will be impossible to view because it stretches too far. that's what the wrap was for. I think what we want to do instead of removing it, is make sure it is centered on the in-between screens ...not 100% sure how to achieve that.

Reply:
@gnunicorn I have fix the issue with adding media query for `min-width 800px`. Attaching screenshot for reference.
![1280](https://user-images.githubusercontent.com/78471590/111666343-a2c0f480-8839-11eb-83ce-99ebcdd1bc6f.PNG)
![860](https://user-images.githubusercontent.com/78471590/111666353-a5bbe500-8839-11eb-8702-be21732ba12a.PNG)